### PR TITLE
Test against PHP 8.0

### PR DESCRIPTION
**Pull request by nicwortel**
See https://github.com/doctrine/migrations/pull/980#issuecomment-736738554, https://github.com/orgs/doctrine/projects/7#card-50248429.

~Blocked by #34~

### CODE REVIEW

**Comment by greg0ire** [comment]
Well… I think you will have to attempt migrating to Github Actions directly:

Screenshot_2020-12-01 doctrine event-manager - Travis CI

**Comment by nicwortel** [comment]
> Well… I think you will have to attempt migrating to Github Actions directly:

I was about to say the same :smile:

No worries, I'll start with that first, then.

**Comment by nicwortel** [comment]
@greg0ire I can probably copy most of the GitHub Actions logic from any repository that is already migrated, but is there some "reference" repository which I should use as the authoritative configuration? https://github.com/doctrine/orm/tree/2.7/.github/workflows for example?

**Comment by greg0ire** [comment]
Yes exactly 👍 You will have to drop Psalm if it is not in use here, and there I already set up the automatic releases workflow, I believe otherwise Github Actions will not show up in this PR, they only seem to show up if there is already at least one Github Action. I'm doing a similar PR on https://github.com/doctrine/lexer/pull/48 as an additional guidance.

File: composer.json [hunk 1]
```diff
@@ -20,12 +20,12 @@
...
     ],
     "require": {
-        "php": "^7.1 || ^8.0"
+        "php": "^7.2 || ^8.0"
     },
     "require-dev": {
         "doctrine/coding-standard": "^6.0",
         "phpstan/phpstan": "^0.12.58",
-        "phpunit/phpunit": "^7.0"
+        "phpunit/phpunit": "^8.0"
```
Comment (greg0ire) [inline composer.json]: You shouldn't drop support for 7.1, and use `^7.0 || ^8.0 || ^9.0` here instead

**Review by alcaeus — APPROVED** [review]

**Comment by greg0ire** [comment]
I dropped the 2 commits that cancel each other out, it looks great now!

**Comment by greg0ire** [comment]
Thanks a lot @nicwortel !

**Pull request merged**